### PR TITLE
add props tables for missing subcomponents

### PR DESCRIPTION
**Pull request by el-dav**
Adding missing props tables for a bunch of subcomponents on the site. I've updated a couple of component prop types to add children to the documentation so the tables wouldn't appear as empty (in the long term we should be consistent about documenting the children prop)

### CODE REVIEW

**Comment by ZarkoPernar** [comment]
`Cmd + Shift + F` Accordian => Accordion 😂

File: site/docs/components/toast/usage.mdx [hunk 1]
```diff
@@ -62,6 +62,10 @@ import { ToastGroup } from "@salt-ds/lab";
...
 <PropsTable componentName="Toast" />
 
-#### ToastGroup
+#### Toast Group
 
 <PropsTable packageName="lab" componentName="ToastGroup" />
+
+#### Toast Content
+
+<PropsTable componentName="ToastContent" />
```
Comment (libertymayc) [inline site/docs/components/toast/usage.mdx:71]:
```suggestion
<PropsTable packageName="lab" componentName="ToastContent" />
```

File: site/docs/components/accordion/usage.mdx [hunk 2]
```diff
@@ -34,4 +34,18 @@ import {
...
+
+<PropsTable componentName="AccordionGroup" />
+
+#### Accordion Header
+
+<PropsTable componentName="AccordionHeader" />
+
+#### Accordion Panel
+
+<PropsTable componentName="AccordionPanel" />
```
Comment (libertymayc) [inline site/docs/components/accordion/usage.mdx:51]:
```suggestion

#### Accordion Group

<PropsTable componentName="AccordionGroup" />

#### Accordion Header

<PropsTable componentName="AccordionHeader" />

#### Accordion Panel

<PropsTable packageName="core" componentName="AccordionPanel" />
```

File: site/docs/components/accordion/usage.mdx [hunk 3]
```diff
@@ -34,4 +34,18 @@ import {
...
+
 <PropsTable componentName="Accordion" />
+
+#### Accordion Group
+
+<PropsTable componentName="AccordionGroup" />
+
+#### Accordion Header
+
+<PropsTable componentName="AccordionHeader" />
```
Comment (libertymayc) [inline site/docs/components/accordion/usage.mdx:47]:
```suggestion
<PropsTable packageName="core" componentName="AccordionHeader" />
```

File: site/docs/components/accordion/usage.mdx [hunk 4]
```diff
@@ -34,4 +34,18 @@ import {
 
 ### Props
 
+#### Accordion
+
 <PropsTable componentName="Accordion" />
+
+#### Accordion Group
+
+<PropsTable componentName="AccordionGroup" />
```
Comment (libertymayc) [inline site/docs/components/accordion/usage.mdx:43]:
```suggestion
<PropsTable packageName="core" componentName="AccordionGroup" />
```

**Comment by lilyvc** [comment]
Could you add for Banner as well? (BannerContent, BannerActions)

File: site/docs/components/toast/usage.mdx (hunk 1 above)
Comment (el-dav) [reply site/docs/components/toast/usage.mdx:71]: It's only `ToastGroup` which is in lab

File: site/docs/components/accordion/usage.mdx (hunk 2 above)
Comment (el-dav) [reply site/docs/components/accordion/usage.mdx:51]: Don't think this is required as core is default

File: site/docs/components/accordion/usage.mdx (hunk 4 above)
Comment (el-dav) [reply site/docs/components/accordion/usage.mdx:43]: Don't think this is required as core is default

**Review by feliciackh — APPROVED** [review]

**Review by lilyvc — APPROVED** [review]

**Pull request merged**